app/test_main.py

```diff
@@ -1,3 +1,37 @@
+import pytest
+
+from typing import List
```

Comment:
I think typing should be higher than pytest, since the latter is a third-party library. By the way, if you are running Python 3.10, you can annotate `list[int]` instead of `typing.List[int]`.